test/e2e/e2e-test/kong/docker/entrypoint.sh

```diff
@@ -18,11 +18,11 @@
 
 # export KONG_NGINX_HTTP_LUA_SHARED_DICT="tracing_buffer 128m"
 
-wget -P ~/ https://github.com/apache/skywalking-nginx-lua/archive/refs/heads/master.zip
+wget -P ~/ -O master.zip https://github.com/apache/skywalking-nginx-lua/archive/39686396ae23d7d341a8ff2212888f02d0b19f6d.zip
 
 unzip -q ~/master.zip -d ~/
 
-cd ~/skywalking-nginx-lua-master
+cd ~/skywalking-nginx-lua-*
```

Comment:
I think it is better to use `git clone`? Such as 
```
# make a new blank repository in the current directory
git init

# add a remote
git remote add origin url://to/source/repository

# fetch a commit (or branch or tag) of interest
# Note: the full history up to this commit will be retrieved unless 
#       you limit it with '--depth=...' or '--shallow-since=...'
git fetch origin <sha1-of-commit-of-interest>
```

Reply:
Downloading resource codes are also very widely used in SkyWalking testing scenario.
Whether am I missing something that you are concern about?

Reply:
I think it doesn't use downloading the tar zip, https://github.com/apache/skywalking/blob/master/test/plugin/agent-test-tools/bin/fetch-code.sh